cves/kernel/CVE-2013-0290.yml

```
@@ -75,7 +78,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [911473]
```

This is not a kernel bug: https://bugzilla.kernel.org/show_bug.cgi?id=911473

It looks like this is from the RedHat bugzilla.

```suggestion
bugs:
  - https://bugzilla.redhat.com/show_bug.cgi?id=911473
```